podvm/qcow2/misc-settings.sh

```diff
@@ -11,6 +11,31 @@
 # This ensures machine-id is generated during first boot and a unique
 # dhcp IP is assigned to the VM
 sudo echo -n > /etc/machine-id
-
 #Lock password for the ssh user (peerpod) to disallow logins
 sudo passwd -l peerpod
+
+# install required packages
+if [ "$CLOUD_PROVIDER" == "vsphere" ]
+then
+# Add vsphere specific commands to execute on remote
+    case $PODVM_DISTRO in
+	centos)
+	    #fallthrough
+	    ;&
+	  rhel)
```

Comment:
extra space

Reply:
Sorry, Snir, which line is the extra space on ?